lib/nuca_backend/hotspots/hotspot.ex

```diff
@@ -43,7 +43,7 @@ defmodule NucaBackend.Hotspots.Hotspot do
       :total_unsterilized_cats,
       :volunteer_id
     ])
-    |> validate_required([:latitude, :longitude, :street_name])
+    |> validate_required([:latitude, :longitude, :address, :description])
```

Comment:
description is not mandatory, so we can remove it from here.

Reply:
ok, i checked the view file which is called when rendered from api call, it's dropping unnecessary fields instead of taking the used ones, so no needed changes there